portfolio/src/main/webapp/comments.js

```diff
@@ -0,0 +1,73 @@
+const api = ""
+
+window.addEventListener("DOMContentLoaded", () => {
+
+    setMessages();
+
+    const send_button = document.getElementById('send-button');
```

Comment:
Very neat, I like the separation of HTML/js and adding event handlers in the script (instead of `onclick=..`).